mlflow/utils/_spark_utils.py

```diff
@@ -0,0 +1,8 @@
+from pyspark.sql import SparkSession
```

Comment:
Oh this is kinda subtle, can we move this line down into the try-catch block below, so that it looks like:

```
def _get_active_spark_session():
    try:
        from pyspark.sql import SparkSession
    except ImportError:
        # Return None if user doesn't have PySpark installed
        return None
    try:
        # getActiveSession() only exists in Spark 3.0 and above
        return SparkSession.builder.getActiveSession()
    except Exception:  # pylint: disable=broad-except
        # Fall back to this internal field for Spark 2.x and below.
        return SparkSession._instantiatedSession
```

That way we handle the case where this method is called but PySpark isn't installed